src/client/client.cpp

```diff
@@ -1060,10 +1062,13 @@ void writePlayerPos(LocalPlayer *myplayer, ClientMap *clientMap, NetworkPacket *
 		[12+12+4+4+4] u8 fov*80
 		[12+12+4+4+4+1] u8 ceil(wanted_range / MAP_BLOCKSIZE)
 		[12+12+4+4+4+1+1] u8 camera_inverted (bool)
+		[12+12+4+4+4+1+1+1] f32 movement_speed
```

Comment:
```suggestion
		[12+12+4+4+4+1+1+4] f32 movement_speed
```

Reply:
hmm, I think the current version is actually correct since it's an offset, not a size